SDK/AppCenter/Microsoft.AppCenter.Windows.Shared/Utils/CultureInfoHelper.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.Globalization;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace Microsoft.AppCenter.Windows.Shared.Utils
+{
+    //https://www.pedrolamas.com/2017/05/03/cultureinfo-changes-in-uwp-part-2/
+    class CultureInfoHelper
+    {
+        // relative location of system locale name
+        private const uint LOCALE_SNAME = 0x0000005c;
+
+        // Constants used by the National Language Support reference
+        private const string LOCALE_NAME_USER_DEFAULT = null;
+        private const string LOCALE_NAME_SYSTEM_DEFAULT = "!x-sys-default-locale";
+
+        private const int BUFFER_SIZE = 530;
+        
+        [DllImport("api-ms-win-core-localization-l1-2-0.dll", CharSet = CharSet.Unicode)]
```

Comment:
Will using this dllimport cause any problems with windows store certification? I think that we should test that for UWP

Reply:
Tested store certification; it failed against both develop and my branch, but both gave the same results, so I don't think there's any _change_ to certifiability at the very least.

Reply:
Can you test certification in a vanilla app that uses the updated sdk? I know puppet doesn't pass, but a vanilla app might. I know there was no diff, but I think that the verification tool stops early when it detects failures, so the diff might not give the whole story here

Reply:
Interestingly, a blank app using the same modules as the puppet app has a few more errors in unmanaged APIs... but the other warnings are the same, and the errors from the blank app are a strict superset of the errors given by the puppet app.  I'd say we're ok though.

Reply:
Passed after changing to release - looks good!